skopt/acquisition.py

```diff
@@ -0,0 +1,203 @@
+import warnings
+
+import numpy as np
+
+from scipy.stats import norm
+
+
+def gaussian_acquisition(X, model, y_opt=None, method="LCB",
```

Comment:
Do we still need this function? Couldnt this be part of the minimize function instead?